neon_diana_utils/helm_charts/neon/neon-iris-websat/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: neon-iris-websat
+description: Deploy Neon IRIS Web Satellite UI
+
+type: application
+version: 0.0.4
```

Comment:
First release should start at `0.0.1

Reply:
Wasn't sure if this was a release for the particular chart or for the whole package, I'll update

Reply:
Each chart (including subcharts) gets to be versioned independently and then we just need to make sure any references to that chart get updated as needed. Only real requirement is to make sure the version gets bumped for every change  to the chart (including dependency changes), otherwise the changes won't make it to our chart repository.